frontend/app/src/components/status/sync/SyncIndicator.vue

```diff
@@ -83,6 +86,9 @@ const actionLogout = async () => {
 
 const performSync = async () => {
   set(pending, true);
+  if (get(syncAction) === SYNC_DOWNLOAD) {
+    set(canRequestData, false);
```

Comment:
To stop some running stuff and postpone the notification for the upgraded account for after the restart.